src/ScriptBuilder/PowerShellCounterWriter.cs

```diff
@@ -16,19 +16,30 @@ public static void WriteScript(string scriptPath, IEnumerable<TimerDefinition> t
                 var slaCounterDefinition = @"New-Object System.Diagnostics.CounterCreationData ""SLA violation countdown"", ""Seconds until the SLA for this endpoint is breached."",  NumberOfItems32";
                 stringBuilder.AppendLine(slaCounterDefinition.PadLeft(slaCounterDefinition.Length + 8));
 
-                foreach (var timer in timers)
+                foreach (var duration in durations)
                 {
-                    var timerDefinition = $@"New-Object System.Diagnostics.CounterCreationData ""{timer.Name}"", ""{timer.Description}"",  NumberOfItems32";
-                    stringBuilder.AppendLine(timerDefinition.PadLeft(timerDefinition.Length + 8));
+                    var durationAverageDefinition = $@"New-Object System.Diagnostics.CounterCreationData ""{duration.Name} Average"", ""{duration.Description}"",  AverageTimer32";
```

Comment:
The new counters should state in the description the unit of measurement, thus in milliseconds. The legacy counters description can probably be updated to to mention that the unit of measurement is seconds.